core-web/libs/data-access/src/lib/dot-workflows-actions/dot-workflows-actions.service.ts

```diff
@@ -53,6 +58,37 @@ export class DotWorkflowsActionsService {
             .pipe(pluck('entity'));
     }
 
+    /**
+     * Returns the workflow actions of the passed url, hostId and language
+     *
+     * @param { host_id: string; language_id: string; url: string; renderMode?: DotRenderMode; } params
```

Comment:
This should be in a new "Actions" service.

Reply:
fixed